bin/node/runtime/src/lib.rs

```diff
@@ -1404,7 +1404,6 @@ impl_runtime_apis! {
 	#[cfg(feature = "try-runtime")]
 	impl frame_try_runtime::TryRuntime<Block> for Runtime {
 		fn on_runtime_upgrade() -> Result<(Weight, Weight), sp_runtime::RuntimeString> {
-			frame_support::debug::RuntimeLogger::init();
```

Comment:
the compilation is failing because debug doesn't exist anymore cc @kianenigma 

I think logger is automatically initialized now, so we can remove no ?

Reply:
Yeah this should be removed, and I think there might be other places in `try-runtime` that we use this the deprecated logger as well. 

I've made an issue here: https://github.com/paritytech/ci_cd/issues/103, I hope we have this in the CI soon.